src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <HarvestStablePackage>false</HarvestStablePackage>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Runtime.WindowsRuntime.csproj">
```

Comment:
Can you please compare the package that gets built with this package against the latest package we released to ensure it continues to support all the same things? In other words if we had an asset it the last package we need an asset in this package, whether via live builds or via harvesting the older assets.